src/expression_utils.jl

```diff
@@ -0,0 +1,172 @@
+#Returns the length of a expression tuple, or 1 if it is not an expression tuple (probably a  Symbol/Numerical).
```

Comment:
Note, this entire file has just been refactored out of `reaction_network.jl` since it encapsulates more generic expression manipulation functions. None of the code is new.